packages/cozy-client/src/CozyClient.js

```diff
@@ -239,6 +239,38 @@ class CozyClient {
     })
   }
 
+  /**
+   * In cozy app context, CozyClient can be instantiated from data injected by the stack
+   *
+   * @param  {string}   selector - Options
+   * @param  {object}   options  - CozyClient constructor options
+   * @returns {object} - CozyClient instance
+   */
+  static fromDOM(selector = '[role=application]', options = {}) {
```

Comment:
I am hesitant on the signature : I think selector will not be used as frequently as the options imho. The problem is that to define options, signature has to be passed, it is a bit inconvenient. I wonder if we should exchange the two args ? It the same problem for fromEnv though.

Reply:
I agree, but I did it this way for consistency. 
I don't have any use case for the option parameter though.

Reply:
Passing the schema, the oauth options, the app name.